public/javascripts/fetch-params.js

```diff
@@ -0,0 +1,26 @@
+const fetchParams = ( method ) => {
+
+const params =
```

Comment:
The indentation is off here, but I would suggest omitting the const variable - it's only used to return on the next line, so may as well do it directly.  Since this makes this a single line function that returns that line, and remembering that we can omit parent for single parameter functions, this can be rewritten as:
```js
const fetchParams = method => ({
  method,
  header: {
    'Accept': 'application/json',
    'Content-Type': 'application/json'
  }
  credentials: 'include'
})
```